src/components/types.ts

```diff
@@ -286,3 +286,31 @@ export interface PDFViewerProps {
   /** The url of the PDF file to be displayed. */
   url: string
 }
+
+export interface Weather {
+  timestamp: number
+  temp: {
+    low: number
+    high: number
+    current?: number
+  }
+  weatherIcon: {
+    icon: string
+    description: string
+  }
+}
+
+export interface WeatherFormat extends DataFormat {
+  /** Array of daily weather data. */
+  weather: Weather[]
+  /** The timezone from which the weather data is from. */
+  location: string
```

Comment:
I don't think this should be time zone. Weather data changes within the city itself. A lot of countries have single timezone. Is this perhaps a typo in the description?